src/utils/options.ts

```diff
@@ -217,12 +216,18 @@ export function duplicateOptionFilter(currentOption: IOption, currentIndex: numb
   return true;
 }
 
-export function formatLabelForSelect(option: IOption, langAsString: IUseLanguage['langAsString']): React.ReactNode {
+export function formatLabelForSelect(
+  option: IOption,
+  langAsString: IUseLanguage['langAsString'],
+  listHasDescription: boolean,
+): React.ReactNode {
   if (option.description) {
     return getParsedLanguageFromText(
       `<b>${langAsString(option.label ?? option.value)}</b><br><span>${langAsString(option.description)}</span>`,
     );
   } else {
-    return getParsedLanguageFromText(`<span>${langAsString(option.label ?? option.value)}</span>`);
+    return getParsedLanguageFromText(
+      `<span style="${listHasDescription && 'font-weight:600'}">${langAsString(option.label ?? option.value)}</span>`,
```

Comment:
I would prefer using a css class from a css module file instead of using inline styling.
```suggestion
      `<span className={listHasDescription && classes.boldOptionLabel}>${langAsString(option.label ?? option.value)}</span>`,
```

Reply:
I see now that we are using `getParsedLanguageFromText` on the whole part of html that is the label and description. Would it be possible to use it on just the `option.label` ?? `option.value` part?

Reply:
Or maybe use `lang()` instead of `langAsString()`, since `lang()` uses `getParsedLanguageFromText()` already

Reply:
I suppose that `getParsedLanguageFromText` is probably not needed at all if you could just return some JSX?

Reply:
> Or maybe use lang() instead of langAsString(), since lang() uses getParsedLanguageFromText() already

Well, since langAsString is used inside of getParsedLanguageFromText, it currently does not format the text string itself

Reply:
Using a css class from a module file instead of inline styling doesn't seem to work 🤔

Reply:
Only if you make it into a react component instead of using the parsing thing

Reply:
I should not have made it like that in the fist place 😅